src/overlays/actors/ovl_En_Ko/z_en_ko.c

```diff
@@ -47,93 +70,1248 @@ static ColliderCylinderInit D_80A9A100 = {
+                              /* ENKO_TYPE_CHILD_1    */ { 0.0f, 0.0f, 0.0f, -20.0f, -10.0f },
+                              /* ENKO_TYPE_CHILD_2    */ { 0.0f, 0.0f, 0.0f, -30.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_3    */ { -10.0f, 10.0f, 10.0f, -10.0f, -30.0f },
+                              /* ENKO_TYPE_CHILD_4    */ { 0.0f, 0.0f, 0.0f, -10.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_5    */ { 0.0f, 0.0f, 0.0f, -20.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_6    */ { 0.0f, 0.0f, 0.0f, -10.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_7    */ { 10.0f, 10.0f, 10.0f, -60.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_8    */ { -10.0f, -10.0f, -20.0f, -30.0f, -30.0f },
+                              /* ENKO_TYPE_CHILD_9    */ { -10.0f, -10.0f, -10.0f, -40.0f, -40.0f },
+                              /* ENKO_TYPE_CHILD_10   */ { 0.0f, 0.0f, 0.0f, -10.0f, -20.0f },
+                              /* ENKO_TYPE_CHILD_11   */ { -10.0f, -10.0f, -20.0f, -30.0f, -30.0f },
+                              /* ENKO_TYPE_CHILD_FADO */ { 0.0f, 0.0f, 0.0f, -20.0f, -20.0f } };
+
+    if (LINK_IS_ADULT && ENKO_TYPE == ENKO_TYPE_CHILD_FADO) {
+        return -20.0f;
+    }
+    return D_80A9A62C[ENKO_TYPE][EnKo_GetForestQuestState(this)];
+}
+
+u8 func_80A97C7C(EnKo* this) {
+    u8 D_80A9A730[13][5] = { /* ENKO_TYPE_CHILD_0    */ { 1, 1, 1, 0, 1 },
+                             /* ENKO_TYPE_CHILD_1    */ { 1, 1, 1, 1, 1 },
+                             /* ENKO_TYPE_CHILD_2    */ { 1, 1, 1, 0, 1 },
+                             /* ENKO_TYPE_CHILD_3    */ { 1, 1, 1, 0, 1 },
+                             /* ENKO_TYPE_CHILD_4    */ { 1, 1, 1, 0, 1 },
+                             /* ENKO_TYPE_CHILD_5    */ { 0, 0, 0, 0, 0 },
+                             /* ENKO_TYPE_CHILD_6    */ { 1, 1, 1, 1, 1 },
+                             /* ENKO_TYPE_CHILD_7    */ { 1, 1, 1, 0, 1 },
+                             /* ENKO_TYPE_CHILD_8    */ { 0, 0, 0, 0, 0 },
+                             /* ENKO_TYPE_CHILD_9    */ { 0, 0, 0, 0, 0 },
+                             /* ENKO_TYPE_CHILD_10   */ { 1, 1, 1, 1, 1 },
+                             /* ENKO_TYPE_CHILD_11   */ { 0, 0, 0, 0, 0 },
+                             /* ENKO_TYPE_CHILD_FADO */ { 1, 1, 1, 1, 1 } };
+
+    return D_80A9A730[ENKO_TYPE][EnKo_GetForestQuestState(this)];
+}
+
+s32 EnKo_IsWithinTalkAngle(EnKo* this) {
+    s16 yawDiff;
+    s16 yawDiffAbs;
+    s32 result;
+
+    yawDiff = this->actor.yawTowardsPlayer - (f32)this->actor.shape.rot.y;
+    yawDiffAbs = ABS(yawDiff);
+
+    if (yawDiffAbs < 0x3FFC) {
+        result = true;
+    } else {
+        result = false;
+    }
+    return result;
+}
+
+s32 func_80A97D68(EnKo* this, GlobalContext* globalCtx) {
+    s16 arg3;
+
+    if (this->unk_1E8.unk_00 != 0) {
+        if ((this->skelAnime.animation == &D_06006A60) == false) {
```

Comment:
```suggestion
        if (this->skelAnime.animation != &D_06006A60) {
```
that doesn't match? same with other `== false` checks

Reply:
non-matching. doing this eliminates an `xor sltiu` pair